docs/source/tools/sandbox.rst

```diff
@@ -226,3 +226,391 @@ Command-line reference
 To start Sandbox, run: ``sandbox [options] <archive>...``.
 
 To see all the available options, run ``daml sandbox --help``.
+
+Metrics
+*******
+
+Enable and configure reporting
+==============================
+
+To enable metrics and configure reporting, you can use the two following CLI options:
+
+- ``--metrics-reporter``: passing a legal value will enable reporting; the accepted values
+  are ``console``, ``csv:</path/to/metrics.csv>`` and ``graphite:<local_server_port>``.
+
+  - ``console``: prints captured metrics on the standard output
+
+  - ``csv:</path/to/metrics.csv>``: saves the captured metrics in CSV format at the specified location
+
+  - ``graphite:<local_server_port>``: sends captured metrics to a local Graphite server. If the port
```

Comment:
```suggestion
  - ``graphite:<server_host>:<server_port>``: sends captured metrics to a Graphite server. If the port
```
Might make more sense to have the server and port separate since the port is optional and has a default value of `2003`.